### PR TITLE
See you in a hundred years

### PR DESCRIPTION
**Pull request by mxaddict**
See you in a hundred years

### CODE REVIEW

File: src/qt/styles/moc_dark.cpp [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/****************************************************************************
```
Comment (aguycalled) [inline src/qt/styles/moc_dark.cpp]: unrelated file

File: src/qt/styles/moc_light.cpp [hunk 2]
```diff
@@ -0,0 +1,89 @@
+/****************************************************************************
```
Comment (aguycalled) [inline src/qt/styles/moc_light.cpp]: unrelated file

File: src/qt/styles/moc_dark.cpp (hunk 1 above)
Comment (mxaddict) [reply src/qt/styles/moc_dark.cpp]: Sorry, removed :)

**Comment by mxaddict** [comment]
Capture
Patch works :)

**Comment by navbuilder** [comment]
A new build of 1479a1a490c469e0297faf863c9b5a4ab77c0826 has completed succesfully!
Binaries available at https://build.nav.community/binaries/100years
